### ISSUE TITLE
3.3V switches off itself, - no issue, more a question!

### ISSUE THREAD

**Issue by schroepfer** [G6EJD/ESP32-2.9-e-Paper-Module#1]
Hello, thank you for your reengeneering!
I have the TTGO T5 2.9 too. When i use a LiPo and use deep sleep in my code, the 3.3V power switches itself off (to 0.9V) after ca. 30 seconds. So the ESP can never wake up. 
I think it is the 3.3V regulator.
Do you know how to prevent this?
BR
Juergen

**Comment by G6EJD** [comment]
The problem is the board is fitted with a power management chip that removes power once it detects very low power consumption from the ESP32. The board is a direct copy of the M5STACK so if you check out the M5 schematic (this board replaces the TFT with an epaper) you will see the management chip, but I have not found a way of disabling it.
This is a design fault that I have reported to Banggood and Lilygo the TTGO supplier and apparently they have produced a new design to resolve this. I would email both Banggood and Lilygo and complain.
If you need help to identify the exact chip I can do that, but I don’t have access to the schematic just now.
Hope this helps.

**Comment by schroepfer** [comment]
Thank you! The chip is the EA3036 http://wiki.wiioe.com/_media/ea3036.pdf .  It has enable-pins.
This is bad. So we can't use it for battery-supply. I will look for the schematic of the M5.....

**Comment by G6EJD** [comment]
Yes, it's a problem. I've uploaded the schematic for you: https://github.com/G6EJD/ESP32-2.9-e-Paper-Module 
LX1 or LX3 supplies 3v3, perhaps disabling (EN pin) one of those will prevent power management and output shutdown, I have not read the datasheet.

**Comment by G6EJD** [comment]
EN (1..3) is essential an output (LX1, 2 or 3) ON/OFF control, so there is no way to stop it undertaking power management and reducing the output when current consumption drops below the threshold. It says 150mA is the threshold, but in practice the ESP32 draws ~50mA but it's not entering low-power mode, so I wonder if it's a programmable current level?

**Comment by schroepfer** [comment]
Thank you for the schematic!
The EA3036 doesn't have any programmable current level! 
What i can see in the Layout, are only R/C combinations on the enable-inputs. It can be, the this is the 30sec. timer when switching off the 3V, after unplug USB.

**Comment by G6EJD** [comment]
The rc combos set the voltage 510K for 3.3v and a C for on delay I think.
